product_docs/docs/ocl_connector/12.1.2.1/index.mdx

```diff
@@ -1,10 +1,9 @@
 ---
 title: EDB OCL Connector
-productStub: true
 directoryDefaults:
   description: "EDB OCL Connector Version 12.1.2.1 Documentation and release notes."
 
-legacyRedirectsGenerated:
+#legacyRedirectsGenerated:
```

Comment:
this is breaking the build, if the intent is to redirect content, it needs to be uncommented. If it needs to be removed, the indented lines below also need to be removed

Reply:
I have removed it, please check